policyengine_us/parameters/calibration/agi_by_source/projections.yaml

```diff
@@ -0,0 +1,167 @@
+description: Historical data and projections of detailed adjusted gross income (AGI) by source (2019-2033)
+metadata: 
+  unit: currency-USD
+  reference:
+  - title: The Budget and Economic Outlook 2023 to 2033 | Congressional Budget Office
+    # Revenue Projections, by Category > Feb 2023 > Sheet 3.Individual Income Tax Details
+    href: https://www.cbo.gov/publication/58848
+total_adjusted_gross_income:
+  2019-01-01: 11_962_000_000_000
+  2020-01-01: 12_650_000_000_000
+  2021-01-01: 14_758_000_000_000
+  2022-01-01: 15_104_000_000_000
+  2023-01-01: 15_514_000_000_000
+  2024-01-01: 16_081_000_000_000
+  2025-01-01: 16_770_000_000_000
+  2026-01-01: 17_463_000_000_000
+  2027-01-01: 18_187_000_000_000
+  2028-01-01: 18_858_000_000_000
+  2029-01-01: 19_496_000_000_000
+  2030-01-01: 20_233_000_000_000
+  2031-01-01: 20_983_000_000_000
+  2032-01-01: 21_767_000_000_000
+  2033-01-01: 22_564_000_000_000
+salaries_and_wages:
```

Comment:
Could we rename these subtitles to their PolicyEngine variable names? If you search the repo they should be findable. For example, salaries and wages in the model are `employment_income`